Makefile

```diff
@@ -171,6 +171,7 @@ test: run_test coverage vg coverage-check vg-check
 # Ignore failure here; it will be detected by Jenkins.
 .PHONY: run_test
 run_test: build_test | $(TEST_OUT_DIR)
+	mkdir -p /var/run/clearwater
```

Comment:
On all the machines I've looked at `/var/run` is `drwxr-xr-x`` and owned by `root:root`. This means if I run my unit tests as a normal (non-root) I can't create this directory. It would be better to try writing to it and if unsuccessful, print a message telling users they need to create it (possibly as a root)